### ISSUE TITLE
Random segfault 

### ISSUE THREAD

**Issue by sbsbs15** [linuxmint/nemo#1022]
Getting random crashes using nemo.

Open in terminal I get this:
(nemo:1116): GLib-GObject-WARNING **: invalid uninstantiatable type '(null)' in cast to 'NemoView'

gdb:
Program received signal SIGSEGV, Segmentation fault.
0x00007ffff50ad9bb in g_type_check_instance_cast ()
from /usr/lib/libgobject-2.0.so.0

I'm using nemo 2.8.5-1 (Archlinux)

**Comment by mtwebster** [comment]
Could I get you to provide a trace?

This can help: https://github.com/linuxmint/Cinnamon/wiki/How-to-use-GDB-to-gather-debug-information-on-a-program

Basically, install debug packages (for nemo, glib and gtk at least) and gdb, then:

```
nemo --quit    (kill nemo)
gdb nemo
> set logging on                         (writes output to a gdb.txt file)
> r                                      (run)
> ..... do stuff to cause crash ...
> thread apply all bt                    (back trace all threads)
> q                                      (quit)
```

And pastebin the contents of the resulting ~/gdb.txt  (feel free to filter any personal/irrelevant lines)

Thanks

**Comment by sbsbs15** [comment]
I do it, but when nemo crash all the system freezes, and I have to change tty and reboot, the resulting file (without back trace because I can't) is this:

Starting program: /usr/bin/nemo 
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
[New Thread 0x7fffe983b700 (LWP 2310)]
[New Thread 0x7fffe903a700 (LWP 2311)]
[New Thread 0x7fffe8839700 (LWP 2312)]
[New Thread 0x7fffdbdf2700 (LWP 2313)]
[New Thread 0x7fffdae71700 (LWP 2315)]
[New Thread 0x7fffda1c7700 (LWP 2316)]
[New Thread 0x7fffd99c6700 (LWP 2317)]
[New Thread 0x7fffcbfff700 (LWP 2318)]
[Thread 0x7fffda1c7700 (LWP 2316) exited]
[Thread 0x7fffd99c6700 (LWP 2317) exited]
[Thread 0x7fffcbfff700 (LWP 2318) exited]
[Thread 0x7fffe8839700 (LWP 2312) exited]
[New Thread 0x7fffe8839700 (LWP 2319)]
[Thread 0x7fffe8839700 (LWP 2319) exited]
[New Thread 0x7fffe8839700 (LWP 2320)]
[New Thread 0x7fffcbfff700 (LWP 2321)]
[New Thread 0x7fffd99c6700 (LWP 2322)]
[Thread 0x7fffd99c6700 (LWP 2322) exited]
[Thread 0x7fffe8839700 (LWP 2320) exited]
[Thread 0x7fffdae71700 (LWP 2315) exited]
[New Thread 0x7fffdae71700 (LWP 2323)]
[New Thread 0x7fffd99c6700 (LWP 2324)]
[Thread 0x7fffd99c6700 (LWP 2324) exited]
[Thread 0x7fffcbfff700 (LWP 2321) exited]
[New Thread 0x7fffcbfff700 (LWP 2325)]
[New Thread 0x7fffd99c6700 (LWP 2326)]
[New Thread 0x7fffe8839700 (LWP 2327)]
[Thread 0x7fffd99c6700 (LWP 2326) exited]
[Thread 0x7fffdae71700 (LWP 2323) exited]
[Thread 0x7fffcbfff700 (LWP 2325) exited]

Program received signal SIGSEGV, Segmentation fault.
0x000000000049d9cc in ?? ()
quit
EOF [assumed Y]

Thank you.

**Comment by danger89** [comment]
Same issue!!
[nemo_core_dump.txt](https://github.com/linuxmint/nemo/files/52656/nemo_core_dump.txt)
[nemo_terminal_output.txt](https://github.com/linuxmint/nemo/files/52655/nemo_terminal_output.txt)

**Comment by danger89** [comment]
Another crash log, using gdb.
[crash_info.txt](https://github.com/linuxmint/nemo/files/52695/crash_info.txt)

**Comment by sbsbs15** [comment]
I think that @mtwebster solved this with f1b593334ea12fcc16bca873fcb1ea058ba76b27 but I'm not sure because the update aren't on arch repos yet.

**Comment by sbsbs15** [comment]
Confirmed, issue solved.
